client/components/date-picker/day.jsx

```diff
@@ -1,87 +1,117 @@
 /**
  * External Dependencies
  */
-var React = require( 'react' ),
-	noop = require( 'lodash/noop' ),
-	classNames = require( 'classnames' );
+import React, { PropTypes, Component } from 'react';
+import { noop, map } from 'lodash';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-var Tooltip = require( 'components/tooltip' );
-
-module.exports = React.createClass( {
-	displayName: 'DatePickerDay',
-
-	propTypes: {
-		date: React.PropTypes.object.isRequired,
-		events: React.PropTypes.array
-	},
-
-	getInitialState: function() {
-		return {
-			showTooltip: false
-		};
-	},
-
-	isPastDay: function( date ) {
-		var today = this.moment().set( {
+import Tooltip from 'components/tooltip';
+import { CalendarEvent } from './event';
+
+class DatePickerDay extends Component {
+	static propTypes = {
+		date: PropTypes.object.isRequired,
+		events: PropTypes.array,
+		maxEventsPerTooltip: PropTypes.number,
```

Comment:
`isPastDay` uses `this.props.moment`, so that should probably be listed as required here.